spec/services/check_employee_association_service_spec.rb

```diff
@@ -0,0 +1,36 @@
+require 'rails_helper'
+
+RSpec.describe CheckEmployeeAssociation, type: :service do
+  let(:user) { create(:user) }
+  let(:company) { create(:company) }
+
+  describe '.call' do
+    subject(:check_association) do
+      described_class.call(*args)
+    end
+
+    context 'when the user not registered on Comarev yet' do
+      let(:args) { [nil, 'not_registered_email', company] }
+
+      it { is_expected.to eq({ message: 'Email sent to new user', status: :ok }) }
+    end
+
+    context 'when the user already exists on Comarev' do
+      describe 'and is not listed as a company employee' do
```

Comment:
I think context fits better here
```suggestion
      context 'and is not listed as a company employee' do
```